README.md

```diff
@@ -162,6 +162,33 @@ you to manage and maintain those images and containers in a production environme
 familiar container cli commands.  For more details, see the
 [Container Tools Guide](https://github.com/containers/buildah/tree/master/docs/containertools).
 
+## [Podman Hello](https://podman.io/images/podman-hello.jpg)
+```
+$ podman run quay.io/podman/hello
+Trying to pull quay.io/podman/hello:latest...
+Getting image source signatures
+Copying blob a6b3126f3807 done  
+Copying config 25c667d086 done  
+Writing manifest to image destination
+Storing signatures
+!... Hello Podman World ...!
+
+         .--"--.           
+       / -     - \         
+      / (O)   (O) \        
+   ~~~| -=(,Y,)=- |         
+    .---. /`  \   |~~      
+ ~/  o  o \~~~~.----. ~~   
+  | =(X)= |~  / (O (O) \   
+   ~~~~~~~  ~| =(Y_)=-  |   
+  ~~~~    ~~~|   U      |~~ 
+
+Project:   https://github.com/containers/podman
+Website:   https://podman.io
+Documents: https://docs.podman.io
+Twitter:   @Podman_io
+```
+
```

Comment:
I've changed to text here but kept a bit of an easter egg to the image in the title.  I'll poke further at the image over time.  I personally like the white on blue, but there's no accounting for my tastes!